README.md

```diff
@@ -1,73 +1,243 @@
+| `#getRequestBodyExamples()` | Retrieve an array of all request body examples that this operation has defined. |
+| `#getRequestBodyMediaTypes()` | Retrieve a list of all the media/content types that the operation can accept a request body payload for. |
+| `#hasRequestBody()` | Determine if this operation has a request body defined. |
+<!-- prettier-ignore-end -->
+
+#### Responses
+
+<!-- prettier-ignore-start -->
+| Method | Description |
+| :--- | :--- |
+| `#getResponseAsJSONSchema()` | Retrive and convert a response on this operation into JSON Schema. |
+| `#getResponseByStatusCode()` | Retrieve the raw response object for a given status code. |
+| `#getResponseExamples()` | Retrieve an array of all response examples that this operation has defined. |
+| `#getResponseStatusCodes()` | Retrieve all status codes that this operation may respond with. |
+| `#hasRequiredRequestBody()` | Determine if this operation has a required request body. |
+<!-- prettier-ignore-end -->
+
+#### Security
+
+<!-- prettier-ignore-start -->
+| Method | Description |
+| :--- | :--- |
+| `#getSecurity()` | Return all security requirements that are applicable for either this operation, or if the operation has none specific to it, then for the entire API. |
+| `#getSecurityWithTypes()` | Return a collection of all security schemes applicable to this operation (using `#getSecurity()`), grouped by how the security should be handled (either AND or OR auth requirements). |
+| `#prepareSecurity` | Return an object of every security scheme that _can_ be used on this operation, indexed by the type of security scheme it is (eg. `Basic`, `OAuth2`, `APIKey`, etc.). |
+<!-- prettier-ignore-end -->
+
+#### Specification Extensions
+
+<!-- prettier-ignore-start -->
+| Method | Description |
+| :--- | :--- |
+| `#getExtension()` | Retrieve a given [specification extension](https://github.com/OAI/OpenAPI-Specification/blob/main/versions/3.1.0.md#specificationExtensions) if it exists on this operation. |
+| `#hasExtension()` | Determine if a given [specification extension](https://github.com/OAI/OpenAPI-Specification/blob/main/versions/3.1.0.md#specificationExtensions) exists on this operation. |
+<!-- prettier-ignore-end -->
+
+### Callbacks
+
+The `Callback` class inherits `Operation` so every API available on instances of `Operation` is available here too. Much like `Operation`, we also support common parameters, summaries, and descriptions that may be set at the [path item level](https://github.com/OAI/OpenAPI-Specification/blob/main/versions/3.1.0.md#pathItemObject) within a `callbacks` definition.
+
+#### General
+
+<!-- prettier-ignore-start -->
+| Method | Description |
+| :--- | :--- |
+| `#getIdentifier()` | Retrieve the primary identifier of this callback. |
+<!-- prettier-ignore-end -->
+
+### Webhooks
+
+Because our `Webhook` class extensions `Operation`, every API that's available on the [Operation](#operation) class is available on webhooks.
```

Comment:
```suggestion
Because our `Webhook` class extends `Operation`, every API that's available on the [Operation](#operation) class is available on webhooks.
```

Reply:
lmao. this is what i get for writing documentation at 1am.